README.md

```diff
@@ -13,71 +12,61 @@ git clone https://github.com/allenai/s2_fos.git
 cd s2_fos
 conda create -y --name s2_fos python==3.8
 conda activate s2_fos
-python setup.py develop
+# Install poetry
+curl -sSL https://install.python-poetry.org | python3 -
+poetry install
 ```
 
 To obtain the necessary data, run these commands after the package is installed:
 
 ```bash
 cd data
-aws s3 cp --no-sign-request s3://ai2-s2-research/s2_fos_artifacts_v001.tar.gz .
-tar -xvzf s2_fos_artifacts_v001.tar.gz
+# Download Langauge identification model from [fasttext](https://fasttext.cc/docs/en/language-identification.html)
+wget https://dl.fbaipublicfiles.com/fasttext/supervised-models/lid.176.bin .
```

Comment:
Do we still need language detection for your model? I guess we haven't tried it with non-English stuff

Reply:
I think we do.